subprojects/docs/src/docs/userguide/getting_started.adoc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[getting_started]]
+= Getting Started
+
+Everyone has to start somewhere and if you're new to Gradle, then this is a good place to begin.
+
+== What is Gradle?
+
+Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool that is designed to be flexible enough to build almost any type of software. It:
+
+ * _Runs on the JVM_ — but it can build anything, including native libraries and executables
+ * _Runs fast_ — particularly with well-designed builds
+ * _Uses conventions and plugins_ — which make it simple to build common types of project
+ * _Is customizable and extensible_
```

Comment:
:o: Feels a little off that this is the only point without extra explanation.

Reply:
This is now in the _What is Gradle?_ chapter/page.